backend/src/controllers/reports/Tab_19_rpt_PA_ActiveProjectsbyPortfolio.js

```diff
@@ -19,7 +19,6 @@ controller.getReport = getReport;
  * @returns {object}
  */
 controller.Tab_19_rpt_PA_ActiveProjectsbyPortfolio = async (request, reply) => {
-  controller.userRequires(request, "PMO-Reports-Capability", reply);
```

Comment:
needs this role added to the route config.role

Reply:
It's in the route already, thanks to you!